assets/sass/patterns/molecules/highlight-item.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../definitions";
+
+.highlight-item {
+  @include constrain-width($max-site-width, "max");
+  @include margin($content-header-padding, "bottom");
+  color: $color-text;
+  list-style-type: none;
+  overflow: hidden;
+  position: relative;
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    @include margin(36, "bottom");
+  }
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+    display: grid;
+    grid-auto-flow: column;
+    grid-column-gap: #{$site-spacing}px;
+    grid-template-columns: 8fr 4fr;
+  }
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--x-wide)}em) {
+    @include margin(0, "bottom");
+    grid-auto-flow: row;
+    grid-column-gap: 0;
+    grid-template-columns: auto;
+  }
+}
+
+.highlight-item:last-child {
+  @include margin(0, "bottom");
+}
+
+.highlight-item__body {
+  width: calc(50% + 12px);
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    width: calc(50% - 12px);
+  }
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+    width: 100%;
+  }
+
+  .author-line {
+    display: none;
+
+    @media only screen and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+      @include padding(12, "top");
+      display: block;
+    }
+  }
+}
+
+.highlight-item__image {
+  width: 100%;
+}
+
+.highlight-item__meta {
+  @include padding(12, "top");
+  color: $color-text-secondary;
+
+  .meta {
+    @include font-size-and-vertical-height(14, 24);
+    color: inherit;
+  }
+
+  .meta__type {
+    &:hover {
+      color: inherit;
+    }
+  }
+
+  .date {
+    @include font-size-and-vertical-height(14, 24);
+  }
+}
+
+.highlight-item__picture {
+  display: inline-block;
+  float: right;
```

Comment:
Can we align this using grid rather than using float?

Reply:
The float is removed and now the style is aligned as grid